Update how-to-discover-sql-existing-project.md (#124636)
Azure/azure-content: articles/migrate/how-to-discover-sql-existing-project.md

         
    - In the manage credentials and discovery sources panel of the Appliance configuration manager, add Domain or SQL Server Authentication credentials that have Sysadmin access on the SQL Server instance and databases to be discovered or have [these permissions](migrate-support-matrix-vmware.md#configure-the-custom-login-for-sql-server-discovery) for each SQL Server instance.
    - Web apps discovery works with both domain and non-domain Windows OS credentials as long as the account used has local admin privileges on servers.
    You can leverage the automatic credential-mapping feature of the appliance, as highlighted [here](./tutorial-discover-vmware.md#start-continuous-discovery).    

> [!Note]

> > For WebApp discovery you may  need to add credentials in DLL format (netbiosdomain\user) for the discovery to complete successfully.

Reviewer: ```suggestion
> For WebApp discovery, you might need to add credentials in DLL format (`netbiosdomain\user`) so that the discovery can complete successfully.
```
